docs/terminal/sydney.md

```diff
@@ -231,7 +231,7 @@ To reduce frequency congestion, several commonly used geographically defined are
 | Sector Name | Lateral Limits | Vertical Limits  |
 | --| ----------------| --------- |
 | City East | Bounded by Rushcutters Bay, Sydney Cricket Ground, Sydney Harbour Bridge South Pylon, Fort Denison, Clark Island, Rushcutters Bay   | Not above `A020`     |
-| CBD | Bounded by Rushcutters Bay, Sydney Cricket Ground, Cleveland Street, Regent Street, George Street, Sydney Harbour Bridge South Pylon, Fort Denison, Clarke Island, Rushcutters Bay | Not above `A020` |
+| CBD | Bounded by Rushcutters Bay, Sydney Cricket Ground, Cleveland Street, Regent Street, George Street, Sydney Harbour Bridge South Pylon, Fort Denison, Clark Island, Rushcutters Bay | Not above `A020` |
```

Comment:
Clarke Island

Reply:
ERSA has a typo, it says Clark Island on the first one, Clarke Island on the second one. Checked google maps and correct spelling is Clark Island

Reply:
Interesting Apple Maps said with an E... but appears you are correct 👍🏻